docs/getting_started/index.rst

```diff
@@ -1,11 +1,46 @@
 Getting started
 ===============
 
+Wagtail is built on the `Django web framework <https://www.djangoproject.com/>`_, so this document assumes you've already got the essentials installed. But if not, those essentials are:
+
+ * `Python <https://wiki.python.org/moin/BeginnersGuide/Download>`_
+ * `pip <https://pip.pypa.io/en/latest/installing.html>`_ (Note that pip is included by default with Python 2.7.9 and later and Python 3.4 and later)
+
+We'd also recommend:
```

Comment:
We're responsible for documenting this software not every prerequisite bit of knowledge, so yes we do require Django/Python understanding as we have to draw the line somewhere. However if you're aware of a suitable tutorial to learn Python from scratch which we can link at the start of this document, let me know, but we definitely shouldn't be documenting any of that ourselves.